src/main/java/org/jabref/gui/entryeditor/LatexReferencesTab.java

```diff
@@ -0,0 +1,109 @@
+package org.jabref.gui.entryeditor;
+
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import javafx.scene.control.ProgressIndicator;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.Tooltip;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.StackPane;
+import javafx.scene.layout.VBox;
+import javafx.scene.text.Text;
+
+import org.jabref.gui.icon.IconTheme;
+import org.jabref.gui.util.TaskExecutor;
+import org.jabref.logic.l10n.Localization;
+import org.jabref.model.database.BibDatabaseContext;
+import org.jabref.model.entry.BibEntry;
+import org.jabref.model.strings.LatexToUnicodeAdapter;
+import org.jabref.model.texparser.Citation;
+import org.jabref.preferences.PreferencesService;
+
+import org.fxmisc.easybind.EasyBind;
+
+public class LatexReferencesTab extends EntryEditorTab {
+
+    private final LatexReferencesTabViewModel viewModel;
+    private ProgressIndicator progressIndicator;
+    private ObservableList<VBox> formattedCitationList;
+    private StackPane searchPane;
+
+    public LatexReferencesTab(BibDatabaseContext databaseContext, PreferencesService preferencesService,
+                              TaskExecutor taskExecutor) {
+        this.viewModel = new LatexReferencesTabViewModel(databaseContext, preferencesService, taskExecutor);
+
+        setText(Localization.lang("LaTeX references"));
+        setTooltip(new Tooltip(Localization.lang("Search this reference in the LaTeX file directory")));
+        setGraphic(IconTheme.JabRefIcons.APPLICATION_TEXSTUDIO.getGraphicNode());
+    }
+
+    private void setupSearchPane() {
+        progressIndicator = new ProgressIndicator();
+        progressIndicator.setMaxSize(100.0, 100.0);
+
+        formattedCitationList = FXCollections.observableArrayList();
+
+        searchPane = new StackPane();
+        searchPane.getStyleClass().add("latexReferences-tab");
+        searchPane.getChildren().add(progressIndicator);
+
+        setContent(searchPane);
+    }
+
+    @Override
+    protected void bindToEntry(BibEntry entry) {
+        setupSearchPane();
+
+        viewModel.setEntry(entry);
+        viewModel.initSearch();
```

Comment:
Bind to entry is also executed when you scroll up and down in the main table and have the entry editor tab open.
So you would likely need to cancel the search when the bib entry changes,

Reply:
Big point, @Siedlerchr. Thank you!